Bottom: 659f893a017442f7b04d79f7e588cc51a7f202ac
Top:    1119721dc13be351235e1a490449324d8a018fe2
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-06 09:50:56 -0500

Refresh of add-new-host-buildbot-worker

---
diff --git a/ansible-playbook/production b/ansible-playbook/production
index 1223c9db..5c572a5a 100644
--- a/ansible-playbook/production
+++ b/ansible-playbook/production
@@ -1,3 +1,5 @@
+builder.novatech-llc.com ansible_host=127.0.0.1 ansible_port=20022 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/bbworker/virtualbox/private_key'
+
 [bacula]
 daedalus.novatech-llc.com
